frontend/src/cards/ui/MultiMapDialog.tsx

```diff
@@ -134,25 +142,9 @@ export function MultiMapDialog(props: MultiMapDialogProps) {
         >
           {/* card heading row */}
           <Grid item xs={12} container justifyContent={'space-between'}>
-            {/* mobile-only close button */}
-            <Grid
-              item
-              xs={12}
-              sx={{ display: { xs: 'flex', sm: 'none' }, mb: 3 }}
-              container
-              justifyContent={'flex-end'}
-            >
-              <Button
-                aria-label="close multiple maps modal"
-                onClick={props.handleClose}
-                color="primary"
-              >
-                <CloseIcon />
-              </Button>
-            </Grid>
             {/* Modal Title */}
             <Grid item xs={12} sm={9} md={10}>
-              <Typography id="modalTitle" variant="h6" component="h2">
+              <Typography id="modalTitle" variant="h6" component="h2" lineHeight={"25px"}>
```

Comment:
can we use the sass variables to stay more consistent across components? general rule of thumb we shouldn't really have any hard coded px values per item if we can avoid it. you can import variables.module.scss and then use `lineHeight={sass.lhLoose}` which should be pretty close to 25px, or if you need to just make a new variable setting like `lhModalHeading: 25px` and use that